### ISSUE TITLE
JSHint must be installed

### ISSUE THREAD

**Issue by rolandoldengarm** [OfficeDev/generator-office#148]
JSHint is not a devDependency, but it is a requirement, otherwise you'll get this error:
````
> gulp vet
[15:04:26] Using gulpfile C:\Projects\git\generator-office\gulpfile.js
[15:04:26] Starting 'vet'...
[15:04:26] Analyzing source with JSHint and JSCS
[15:04:26] 'vet' errored after 108 ms
[15:04:26] Error: Cannot find module 'jshint/src/cli'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\Projects\git\generator-office\node_modules\gulp-jshint\src\extract.js:1:79)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
````
The current version prefers globally, so probably not add it as devDependency. It does work though, I added it locally.
But this should be documented, because I did not know this.

So, document it or add as devDependency?

**Comment by gavinbarron** [comment]
I installed jshint globally and gulp-jshint locally and got the same error :|

After installing the jshint locally the error was resolved.

**Comment by beth-panx** [comment]
closed by pr #186
